src/chaplin/views/view.coffee

```diff
@@ -194,7 +194,7 @@ module.exports = class View extends Backbone.View
   #   e.g.
   #   @delegate('click', 'button.confirm', @confirm)
   delegate: (eventName, second, third) ->
-    if Backbone.utils
+    if Backbone.utils and Backbone.utils?.delegate
```

Comment:
I will stick to your recommendation. I will provide a patch for NativeViews.

Exoskeleton runs faster on low-memory [Firefox OS](https://www.mozilla.org/en-US/firefox/os/) devices.